lib/src/dice_roller.dart

```diff
@@ -78,7 +84,10 @@ class DiceRoller with LoggingMixin {
     return RollResult(
       expression: '${ndice}d$sideVals',
       opType: OpType.rollVals,
-      metadata: RollMetadata(rolled: results),
+      metadata: RollMetadata(
+        rolled: results,
+        diceUsed: {'dF': ndice}
```

Comment:
despite my incorrect copy-paste comment at the top of this method, this roll is for syntax like `2d[2,3,5,7]`